storage/innobase/row/row0mysql.cc

```diff
@@ -1689,7 +1689,8 @@ init_fts_doc_id_for_ref(
 
 		ut_ad(foreign->foreign_table != NULL);
 
-		if (foreign->foreign_table->fts != NULL) {
+		if (foreign->foreign_table->space
+		    && foreign->foreign_table->fts != NULL) {
```

Comment:
We’re comparing both pointers to `nullptr`, one implicitly and the other explicitly. I’d remove the `!= NULL`.